source/items/platearmor.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Plate Armor
+layout: default
+---
+# Plate Armor
+
+{% comment %}
```

Comment:
Why is the `{% comment %}` and `{% endcomment %}` here?

Reply:
Was originally intended to make it easier to add an image later, but I couldn't get a decent screenshot.  Just removed the section instead.

Reply:
wait...you're using _screenshots_ to get the item pics?